HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -889,6 +889,137 @@ copymethod:: SimpleNumber -atanh
 copymethod:: SimpleNumber -owensT
 
 
+subsection:: Testing
+
+method:: ==
+
+Overridden by the need to check for equality of each element with its integer index as independent of identity. See link::Classes/Object#Equality and Identity::.
```

Comment:
"Overridden by the need to check" is not quite right. 
I think this is too detailed.
Better, probably:

Returns true if each element is equal with its corresponding element in the receiver. See link::Classes/Object#Equality and Identity::.

Reply:
I corrected it as you suggested.

p.s.:
I do not remember why I wrote this as I wrote this PR almost 6 months ago.
"Overridden by the need to check" seems to be come from "Whenever == is overridden in a class" in the following parts:
- https://github.com/supercollider/supercollider/blob/66d2978268be620935e4569e000836794b7f8448/HelpSource/Classes/Object.schelp#L209
- https://github.com/supercollider/supercollider/blob/66d2978268be620935e4569e000836794b7f8448/HelpSource/Classes/Object.schelp#L353
- https://github.com/supercollider/supercollider/blob/66d2978268be620935e4569e000836794b7f8448/HelpSource/Classes/Object.schelp#L363